engines/grim/actor.h

```diff
@@ -538,8 +539,8 @@ class Actor : public PoolObject<Actor> {
 	float getGlobalAlpha() const { return _globalAlpha; }
 	AlphaMode getAlphaMode() const { return _alphaMode; }
 	float getEffectiveAlpha() const { return _alphaMode != AlphaOff ? _globalAlpha : 1.f; }
-	void setGlobalAlpha(float alpha) { _globalAlpha = alpha; }
-	void setAlphaMode(AlphaMode mode) { _alphaMode = mode; }
+	void setGlobalAlpha(float alpha, const char* mesh);
```

Comment:
Good point! I changed the function signatures accordingly. Please see the additional commit (should be squashed before the final merge).